src/preprocessor-unpickle-reference.py

```diff
@@ -0,0 +1,45 @@
+import sys
+import pandas as pd
+import numpy as np
+import click
+import matplotlib.pyplot as plt
+import altair as alt
+import dataframe_image as dfi
+from sklearn.preprocessing import (
+    OneHotEncoder,
+    OrdinalEncoder,
+    StandardScaler,
+    FunctionTransformer,
+    
+)
+import pickle
```

Comment:
What version of pickle are you using?
Let me know so I can add it to environment

Reply:
Sorry didnt read the previous description. Resolved.